common/include/common/config/GameConfig.h

```diff
@@ -82,6 +82,8 @@ struct GameConfig
     CfgVar<bool> glares = true;
     CfgVar<bool> show_enemy_bullets = true;
     CfgVar<bool> linear_pitch = false;
+    CfgVar<bool> multi_hitsounds = true;
```

Comment:
Add some description in README about those modes because i'm not sure how this feature works currently